template/READ_BEFORE_COMMIT.md.tpl

```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+This document lays out the next steps for AWS service teams upon bootstrapping the ACK {{ .ServicePackageName }}-controller repository successfully.
+
+1. To get started, look inside the `generator.yaml` file, and remove the resource(s) from the ignore list and execute `make build-controller` from the ACK code-generator.
+2. Add any custom code inside the `templates` directory.
+3. Refer to testing documentation to add e2e test inside the `test` directory.
```

Comment:
There should be a link to this testing documentation here.
@RedbackThomson do you know if we have documentation on how to write/structure python e2e tests?
I found this [one](https://aws-controllers-k8s.github.io/community/docs/contributor-docs/testing/) which talk about how to execute e2e tests.

Reply:
Nothing on the public internet. I have notes that I've kept since working with Bruce on the RDS team which would be good to publish